CONTRIBUTION.md

```diff
@@ -166,23 +166,10 @@ storiesOf('Components|Molecules/Top content bar', module)
 
 ## How-to release a component
 
-**Do not use the release scripts: ~~`npm run release`~~ or ~~`npm run prerelease`~~**
-
 - Ensure you have an npm account under the `@axa-ch` organisation.
 - Run `npm whoami` and in case of you not being logged in: `npm login`
-- Create a new _release branch_ that follows this pattern: `release/<component-name>`
-- Clean the main and components dependencies:
-  - `npx lerna clean`
-  - `rm -rf node_modules`
-- Run `npm install`.
-- Manually add your component to `lerna.json` and remove all not-to-be-released components from the same file.
-- Commit and push your lerna.json changes to the release branch.
-- Ensure you have the necessary credentials to push to the Githup repo via commandline commands (the following lerna commands do this automatically).
-- Run `npm run test && npx lerna version && npx lerna publish from-package`
-- Lerna is going to ask you how to update the changed packages. Follow the instructions and keep in mind the semver rules: https://semver.org/
-- Lerna will update the component `package.json` with new version numbers and auto-commit/push those.
-- Manually undo the removal of all not-to-be-released components in lerna.json, run `npm install` and commit the generated package-lock files to the release branch.
-- Merge the release branch to the `develop` branch in order to preserve the new version numbers.
+- Ensure you have admin rights on github
+- Switch to branch `develop` and run `npm run release`. Follow the guidelines in the CLI
```

Comment:
If you do that, another requirement would be to have admin rights on our repo. We should at least mention that, if this is the workflow. Otherwise we should open develop for direct-pushes.

Reply:
i did write it line 171